### PR TITLE
Add User service and favorite and comments

### PR DESCRIPTION
**Pull request by remy-rm**

### CODE REVIEW

**Comment by remy-rm** [comment]
adding the service user 
-refactoring the allrecipes page code to include the recipesDisplay component 
-added favorite pages
-addition of boutton
-implementation of comments
-added FontAsworme library

File: src/app/services/users/users.service.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      this.favoriteRecipes.splice(index, 1);
+    }
+    this.favoriteUpdateSubject.next();
+  }
+
+  isActive(id: number): boolean {
+    return this.favoriteRecipes.indexOf(id) !== -1;
+  }
+
+  getLatestCommentsByRecipeId(recipeId: number, count: number): string[] {
```
Comment (Benja-san) [inline src/app/services/users/users.service.ts:39]: If you just want to sort the comments for one recipe in order desc you can optimize it with : array.sort et array.reverse

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/component/recipesdisplay/recipesdisplay.component.scss
Comment (ArPimentel) [inline src/app/component/recipesdisplay/recipesdisplay.component.scss:1]: I can't find the life I class allrecipes container--card of allrecipes in (recipe-display); You can modify moveLeft and moveRight for better fluency in class="allrecipes_container--card" with transition: transform 0.40s ease-in-out?

File: src/app/component/recipesdisplay/recipesdisplay.component.ts
Comment (ArPimentel) [inline src/app/component/recipesdisplay/recipesdisplay.component.ts:1]: Can you change translateX Value to 200 or 300?

**Review by ArPimentel — CHANGES_REQUESTED** [review]
Change request

**Review by ArPimentel — APPROVED** [review]
Good

**Review by Benja-san — APPROVED** [review]
Good Job !

**Pull request merged**